docs/DeveloperGuide.md

```diff
@@ -629,7 +629,7 @@ testers are expected to do more *exploratory* testing.
    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
 
    1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+      Expected: Delete popup shows up. After confirming on the popup, first contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
```

Comment:
Do we still have a timestamp in the status bar? Is this some old implementation from AB3? (I might be missing it cause I never noticed a timestamp). If we do not have the timestamp anymore should we delete the last part?